src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  deleteTodo: (id: number) => void;
+  isDeleting: boolean,
+  deletedId: number,
```

Comment:
think about making context for these states to avoid passing them through TodoList without using